cranelift/codegen/src/isa/x86/abi.rs

```diff
@@ -796,6 +885,55 @@ fn insert_common_prologue(
         }
     }
 
+    // Now that RSP is prepared for the function, we can use stack slots:
+    if csrs.iter(FPR).len() != 0 {
+        let mut fpr_offset = 0;
```

Comment:
nit: can you move this definition just before its first use, above the for loop at line 904?